src/components/RelatiesKoppelingen/RelatiesKoppelingen.js

```diff
@@ -150,9 +162,9 @@ const RelatiesKoppelingen = ({
                 })
 
         /**
-         * Function to inialize the needed data for the object of type 'Beleidskeuze'
-         * Gets the relations from and to the object, and gets the active verordeningsStructure
-         * When data is set in State we set the loading state to False
+         * Function to set the intitialized data for a Beleidskeuze.
```

Comment:
Why was the existing documentation replaced?

Reply:
I wanted to sum up what has been described, but have reverted the changes.